.github/workflows/publishbot.yml

```diff
@@ -29,10 +29,10 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - uses: ./.github/actions/setup
       - run: |
           apt update
           apt install -y build-essential
+      - uses: ./.github/actions/setup
```

Comment:
`npm ci` tries to install [`node-gyp`](https://github.com/nodejs/node-gyp#on-unix), which fails because `make` is missing. I mistakenly installed `build-essential` after the setup step...